client/sites-dashboard/components/sites-container.tsx

```diff
@@ -0,0 +1,138 @@
+import { ClassNames } from '@emotion/react';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import EmptyContent from 'calypso/components/empty-content';
+import { SiteData } from 'calypso/state/ui/selectors/site-data';
+import { SitesTable } from './sites-table';
+
+const EmptySites = styled( EmptyContent )`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+`;
+
+const SecondaryText = styled.p`
+	max-width: 650px;
+`;
+
+const Title = styled.div`
```

Comment:
@zaguiini What i did for now to kind of center states that doesn't have an illustration is to give the title a 50% margin from the top.